rockcraft.yaml

```diff
@@ -5,52 +5,66 @@ base: ubuntu:22.04 # the base environment for this ROCK
 version: "5.0.14-12" # just for humans. Semantic versioning is recommended
 summary: MongoDB in a rock. # 79 char long summary
 description: |
-    MongoDB is a source-available cross-platform
-    document-oriented database program. Classified
-    as a NoSQL database program, MongoDB uses JSON
-    -like documents with optional schemas.
+  MongoDB is a source-available cross-platform
+  document-oriented database program. Classified
+  as a NoSQL database program, MongoDB uses JSON
+  -like documents with optional schemas.
 license: Apache-2.0 # your application's SPDX license
 entrypoint:
-    - /usr/bin/setpriv
-    - --clear-groups
-    - --reuid
-    - mongodb
-    - --regid
-    - mongodb
-    - --
-    - /usr/bin/mongod
+  - /usr/bin/setpriv
+  - --clear-groups
+  - --reuid
+  - mongodb
+  - --regid
+  - mongodb
+  - --
+  - /usr/bin/mongod
 
 platforms: # The platforms this ROCK should be built on and run on
-    amd64:
+  amd64:
 
 parts:
-    mongo-snap:
-        plugin: nil
-        stage-snaps:
-            - charmed-mongodb/5/edge
-        overlay-packages:
-            - ca-certificates
-    dep-debs:
-        plugin: nil
-        stage-packages:
-            - util-linux
-            - libssh-4
-            - libbrotli1
-    non-root-user:
-        plugin: nil
-        after: [mongo-snap]
-        overlay-script: |
-            # Create a user in the $CRAFT_OVERLAY chroot
-            groupadd -R $CRAFT_OVERLAY -g  584788 mongodb
-            useradd -R $CRAFT_OVERLAY -M -r -g mongodb -u 584788 mongodb
-        override-prime: |
-            craftctl default
-            # Give permission and create the required directories
-            mkdir -p $CRAFT_PRIME/data/db
-            chmod 0755 $CRAFT_PRIME/data/db
-            chown -R 584788:584788 $CRAFT_PRIME/data/db
-    rock-license:
-        plugin: dump
-        source: licenses
-        organize:
-            LICENSE-rock: licenses/LICENSE-rock
+  mongo-snap:
+    plugin: nil
+    stage-snaps:
+      - charmed-mongodb/5/edge
+    overlay-packages:
+      - ca-certificates
+  dep-debs:
+    plugin: nil
+    stage-packages:
+      - libssh-4
+      - libbrotli1
+  non-root-user:
+    plugin: nil
+    after: [mongo-snap]
+    overlay-script: |
+      # Create a user in the $CRAFT_OVERLAY chroot
+      groupadd -R $CRAFT_OVERLAY -g  584788 mongodb
+      useradd -R $CRAFT_OVERLAY -M -r -g mongodb -u 584788 mongodb
+    override-prime: |
+      craftctl default
+      # Give permission and create the required directories
+      mkdir -p $CRAFT_PRIME/data/db
+      chmod 0755 $CRAFT_PRIME/data/db
+      chown -R 584788:584788 $CRAFT_PRIME/data/db
+      # enable security monitoring
+      rocks=usr/share/rocks/
+      mkdir -p ${rocks}
+      ## for deb packages
```

Comment:
nit can we have enters after the comments

Reply:
I can try but linter was complaining about those

Reply:
hmm maybe we can copy the linter from opensearch? they seem to have less problems, and then you could have the dpkg command the way you originally intended?

Reply:
looks like opensearch rock does not have a linter